github-management/kubernetes-repositories.md

```diff
@@ -127,6 +130,62 @@ ecosystem. Carries the endorsement of the Kubernetes community.
      Kubernetes steering committee elections. in the Kubernetes community
    * Repository must be approved by SIG-Architecture
 
+## Creating Repositories
+
+### Non-staging repositories
+
+For non-staging repositories, suggestions on how to create
+a new repository are described below.
```

Comment:
Is using the word "suggestions" enough to show that these are not hard guidelines - there could be corner cases that might require slight deviations from this list. We could probably add these corner cases as we come across them but I want to make sure that (appropriate) slight deviations don't get flagged as "not following the process".